autogen/gradio_gui/README.md

```diff
@@ -0,0 +1,104 @@
+# AutoGen's Gradio GUI
+
+This is a GUI for AutoGen. It is written in Python and uses the [Gradio](https://gradio.app/) library.
+
+## ⭐Installation
+
+1. install AutoGen:
+
+```bash
+python -m pip install pyautogen
+```
+
+## ⭐Run
+
+1. create `OAI_CONFIG_LIST`, write your OpenAI API key or azure API key. For example:
+
+```
+[
+    {
+        "model": "gpt-3.5-turbo-16k",
+        "api_key": "----------------------------------",
+        "api_base": "https://--------.openai.azure.com",
```

Comment:
This needs to be updated when v0.2 is released.

Reply:
sorry,  v0.2 of what

Reply:
v0.2 of pyautogen, no worries, just a reminder.